QoL/Modules/NPCSellAll.cs

```diff
@@ -16,8 +17,18 @@ public class NPCSellAll : FauxMod
         [SerializeToSetting]
         public static bool JinnSellAll = true;
 
+        [SerializeToSetting]
+        public static bool NailsmithBuyAll = true;
+
         private static readonly int[] RELIC_COST = { 200, 450, 800, 1200 };
         private const int EGG_COST = 450;
+        private static readonly (int ore, int geo)[] NAIL_UPGRADE_COST =
```

Comment:
small nit: I'd put this before the EGG_COST or move the EGG_COST up so that the arrays are together